src/qt/rpcconsole.cpp

```diff
@@ -311,29 +312,80 @@ void RPCConsole::setClientModel(ClientModel *model)
         ui->peerWidget->setColumnWidth(PeerTableModel::Address, ADDRESS_COLUMN_WIDTH);
         ui->peerWidget->setColumnWidth(PeerTableModel::Subversion, SUBVERSION_COLUMN_WIDTH);
         ui->peerWidget->setColumnWidth(PeerTableModel::Ping, PING_COLUMN_WIDTH);
+        ui->peerWidget->horizontalHeader()->setStretchLastSection(true);
 
-        // create context menu actions
+        // create peer table context menu actions
         QAction* disconnectAction = new QAction(tr("&Disconnect Node"), this);
-
-        // create context menu
-        contextMenu = new QMenu();
-        contextMenu->addAction(disconnectAction);
-
-        // context menu signals
-        connect(ui->peerWidget, SIGNAL(customContextMenuRequested(const QPoint&)), this, SLOT(showMenu(const QPoint&)));
+        QAction* banAction1h      = new QAction(tr("&Ban Node for") + " " + tr("&1 hour"), this);
```

Comment:
I rechecked the code and during a translation for my build I came to this.
The ``&`` is used as a shortcut when pressing ALT IMHO, so this will produce 2 shortcuts for every single menu entry. I'd suggest just removing them, because it will be hard to make them unique and still usable.